tests/test_math_ops_coverage.py

```
@@ -0,0 +1,854 @@
+
+        ##################################################################################
+        # Test any/all edge cases
+        ##################################################################################
+        # Test any with no shape (line 1656-1657)
+        a = Scalar(1.)
+        b = a.any()
+        self.assertTrue(b)
+
+        # Test any with builtins (line 1670-1674)
+        a = Boolean([False, True, False])
+        Qube.prefer_builtins(True)
```

Ruff (0.14.7)

Boolean positional value in function call

(FBT003)